docs/spherical-video-v2-rfc.md

```diff
@@ -234,6 +236,184 @@ aligned(8) class EquirectangularProjection ProjectionDataBox(‘equi’, 0, 0) {
+}
+```
+
+##### Semantics
+
+- `reserved` are fields where all bits are set to 0. A MeshProjection box
+    version change is required if any of these bits are allowed to be set to 1
+    in a future revision of the spec.
+- `coordinate_count` is the number of floating point values used in the
+    vertices.
+- `coordinate` is a floating point value used in mesh vertices.
+
+- `vertex_count` is the number of position (x,y,z) & texture coordinate (u,v)
+    pairings used in the projection mesh.
+- `ccsb` coordinate count size in bits = ceil(log2(coordinate_count * 2))
+- `x_index_delta` is a delta from the previous x_index into the
+    list of coordinates. For the first element, the previous index is assumed to
+    be zero. These integers are encoded in a zig-zag scheme, similar to
+    [Protocol Buffers's signed integers]
+    (https://developers.google.com/protocol-buffers/docs/encoding#signed-integers).
+    An integer `n` that is greater than or equal to 0 is encoded as `n * 2`. An
+    integer `n` that is less than 0 is encoded as `-n * 2 - 1`.
+- `y_index_delta` is a delta from the previous y_index and has the same encoding
+    and initial index as `x_index_delta`
+- `z_index_delta` is a delta from the previous z_index and has the same encoding
+    and initial index as `x_index_delta`
+- `u_index_delta` is a delta from the previous u_index and has the same encoding
+    and initial index as `x_index_delta`
+- `v_index_delta` is a delta from the previous v_index and has the same encoding
+    and initial index as `x_index_delta`
+
+- `padding` contains 0-7 bits to align to the next byte boundary.
+
+- `vertex_list_count` is the number of vertex index lists that describe the
+    projection mesh.
+- `texture_id` is the Texture ID the UV coordinates refer to.
+    *   0 for video frames in this track.
+    *   >0 reserved
+- `index_type` specifies what the indices refer to. The valid values are:
+    *   0: Triangles
+    *   1: Triangle Strip
+    *   2: Triangle Fan
+- `index_count` is the number of vertex indices in this vertex list.
+- `vcsb` vertex count size in bits = ceil(log2(vertex_count * 2))
+- `index_as_delta` is a delta from the previous index into the
+    list of unique vertices. For the first element, the previous index is
+    assumed to be 0. These integers are encoded in a zig-zag scheme, similar to
+    [Protocol Buffers's signed integers]
+    (https://developers.google.com/protocol-buffers/docs/encoding#signed-integers).
+    An integer `n` that is greater than or equal to 0 is encoded as `n * 2`. An
+    integer `n` that is less than 0 is encoded as `-n * 2 - 1`.
+
+##### Notes:
+
+* All fields are big-endian msbf.
+* Parsers should ignore boxes they don't know about.
+* Parsers should ignore extra bytes at the end of a box.
+* Boxes / fields may be added to the end of a box without incrementing the
+version number.
+* (x,y,z) coordinates are expressed in an OpenGL-style right-handed coordinate
```

Comment:
Since a coordinate system is defined, I think it would be useful to define (or at least suggest) triangle winding as well.